include/circt/Dialect/Ibis/IbisOps.td

```diff
@@ -632,11 +627,8 @@ def ContainerOp : IbisOp<"container", [
     llvm::StringRef getModuleName() {
       return getModuleNameAttr();
     }
-  }];
-
-  let extraClassDefinition = [{
     // For hw:InnerSymbol - the symbol targets the operation and not any result.
-    std::optional<size_t> $cppClass::getTargetResultIndex() {
+    static std::optional<size_t> getTargetResultIndex() {
```

Comment:
I don't understand how this works without the `$cppClass`.

Reply:
This is inside `extraClassDeclaration` so it'll get emitted under the class definition, in the generated `.h` file.